### PR TITLE
Support for sorting in UI

### PR DESCRIPTION
**Pull request by FBRTMaka**
@oceanzus , @DanielJMaher , @c0mster   Please review!

### CODE REVIEW

File: ooiservices/app/uframe/assetController.py [hunk 1]
```diff
@@ -233,7 +233,14 @@ def _associate_events(id):
...
 
-    result = sorted(result, key=itemgetter('eventId'))
+    try:
+        if request.args.get('sort') and request.args.get('sort') != "":
+            sort_by = request.args.get('sort')
+        else:
+            sort_by = 'eventId'
+        result = sorted(result, key=itemgetter(sort_by))
+    except (TypeError, KeyError) as e:
+        print e
```
Comment (oceanzus) [inline ooiservices/app/uframe/assetController.py]: maybe use raise so it gets to the logs?

Comment (FBRTMaka) [inline ooiservices/app/uframe/assetController.py]: @oceanzus good call

**Comment by birdage** [comment]
@FBRTMaka this ready to go?

**Comment by FBRTMaka** [comment]
@birdage , yup! ready to go!

**Pull request merged**